src/components/ItemsList.js

```diff
@@ -43,43 +44,54 @@ const ItemsList = () => {
     history.push('/additem');
   };
 
-
   return (
     <div className="items-list">
       <h1>Your Shopping List</h1>
       {loading && <p>Loading...</p>}
       {error && <p>An error has occured...</p>}
       {shoppingList && !shoppingList.length && (
-            <div className="add-item">
-              <p>You haven't created a shopping list yet...</p>
-              <button type="submit" onClick={handleRedirect}>
-                Add First Item
-              </button>
-            </div>
-          )}
-      <form>
+        <div className="add-item">
+          <p>You haven't created a shopping list yet...</p>
+          <button type="submit" onClick={handleRedirect}>
+            Add First Item
+          </button>
+        </div>
+      )}
+      <div className="search-list">
+        {shoppingList && shoppingList[0] ? (
```

Comment:
This is totally up to you all, and I'm not even sure if it would actually be more beneficial to what's already there or not, but I wonder if it might be an idea to add in variable names for these different conditions 

e.g. 
const listHasAtLeastOneItem = shoppingList && shoppingList[0]
const listHasNoItems = shoppingList && !shoppingList.length

Reply:
Thanks Alex, that makes a lot of sense and definitely makes the code more readable, I've added those constants in!